cdap-docs/deploy.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+################################################################################
+### Deploy script for docs
+# Deploys zip files created by build scripts
+# First parameter is cdap (default)
+# Second parameter is version
+# third parameter is the remote directory to be created on the web/doc server
+################################################################################
+DEBUG=${DEBUG:-no}
+PROJECT=${1:-cdap}
+PROJECT_DOCS=${PROJECT}-docs
+#PROJECT_PATH=
+VERSION=${2:-2.8.1}
+PROJECT_VERSION=${3:-2.8.1}
+RSYNC_OPTS='-a --human-readable --progress --stats --rsync-path="sudo rsync"'
+WEB_FILE=${PROJECT}-docs-${VERSION}-web.zip
+GITHUB_FILE=${PROJECT}-docs-${VERSION}-github.zip
+JOB_DIR=/var/bamboo/xml-data/build-dir/CDAP-DRBD-JOB1
```

Comment:
this can be passed in using built-in bamboo variables ( ``bamboo.buildResultKey`` I believe).  Useful in case the build name changes